.github/workflows/site.yml

```diff
@@ -109,10 +109,17 @@ jobs:
           LINK="https://${{env.AWS_BUCKET_NAME}}.s3.${{env.AWS_REGION}}.amazonaws.com"
           aws s3 cp "$SITE" "s3://${{env.AWS_BUCKET_NAME}}/$FOLDER/" --recursive
           echo "$LINK/$FOLDER/index.html" > message
+          cd checkstyle
+          ./.ci/generate-extra-site-links.sh ${{needs.parse_pr_info.outputs.commit_sha}} \
+            "$LINK/$FOLDER"
```

Comment:
Question: if we do have issues with our new site link generation, we are able to remove call to script and keep everything else in this file the same, right?

Reply:
Yes, it should not be a problem. Here is a test when these lines are removed:
```
          cd checkstyle
          ./.ci/generate-extra-site-links.sh ${{needs.parse_pr_info.outputs.commit_sha}} \
            "$LINK/$FOLDER"
```
https://github.com/stoyanK7/checkstyle/actions/runs/4552842049
https://github.com/stoyanK7/checkstyle/pull/62#issuecomment-1488323191
https://github.com/stoyanK7/checkstyle/blob/582fe5bf588c77b1f4dcddd045840c0506a1c21e/.github/workflows/site.yml#L93-L94
![Screenshot from 2023-03-29 12-17-19](https://user-images.githubusercontent.com/23459549/228503605-687d3d24-2543-4f04-9dcd-ef791e812a91.png)